cogs/moderation.py

```diff
@@ -293,6 +298,54 @@ async def clear(self, ctx: Context, amount: int) -> None:
         await asyncio.sleep(2.5)
         await message.delete()
 
+    @command()
+    @has_permissions(manage_messages=True)
+    async def shift(self, ctx, count: int, target: TextChannel, copy: bool = False) -> None:
+        """Copy or Move specified messages amount to specified channel"""
+        messages = []
+        async for message in ctx.message.channel.history(limit=count):
+            embed = Embed(description=message.content, color=Color.green())
+            embed.set_author(name=message.author.name, icon_url=message.author.avatar_url)
+            embed.timestamp = message.created_at
+            messages.append(embed)
+
+            if not copy:
+                await message.delete()
+
+        await target.send(f'Message source : {ctx.message.channel.mention}.')
```

Comment:
```suggestion
        await target.send(f'Message source : {ctx.channel.mention}.')
```

Reply:
You should also move the `:` there shouldn't be a space before it:
```suggestion
        await target.send(f'Message source: {ctx.message.channel.mention}.')
```